components/admin_console/billing/billing_subscriptions.tsx

```diff
@@ -200,10 +249,25 @@ const BillingSubscriptions: React.FC<Props> = () => {
                             onDismiss={() => handleHide()}
                         />
                     )}
-                    <div
-                        className='BillingSubscriptions__topWrapper'
-                        style={{marginTop: '20px'}}
-                    >
+                    {showCreditCardBanner && (
+                        <AlertBanner
+                            mode='info'
+                            title={
+                                <FormattedMessage
+                                    id='admin.billing.payment_info.creditCardAboutToExpire'
+                                    defaultMessage='Your credit card is about to expire'
```

Comment:
Would this be `your credit card has expired` given that we are checking with today's date directly?